client/asset/dcr/dcr.go

```diff
@@ -1996,15 +1996,12 @@ func (dcr *ExchangeWallet) ValidateSecret(secret, secretHash []byte) bool {
 // first checking for a unspent output, and if not found, searching indexed
 // wallet transactions.
 func (dcr *ExchangeWallet) Confirmations(ctx context.Context, id dex.Bytes) (confs uint32, spent bool, err error) {
-	// Could check with gettransaction first, figure out the tree, and look for a
-	// redeem script with listscripts, but the listunspent entry has all the
-	// necessary fields already.
```

Comment:
I believe this is an outdated comment? Don't see a need for redeem script in this method.